package.json

```diff
@@ -33,10 +33,10 @@
     "@angular/platform-browser": "^18.0.0",
     "@angular/platform-browser-dynamic": "^18.0.0",
     "@angular/router": "^18.0.0",
-    "@piwikpro/tracking-base-library": "^1.1.2",
     "rxjs": "~7.8.1",
     "tslib": "^2.6.2",
-    "zone.js": "^0.14.6"
+    "zone.js": "^0.14.6",
+    "@piwikpro/tracking-base-library": "^1.1.2"
```

Comment:
Question: why do we need base library in the root `package.json`?

Reply:
added "workspaces" entry in the package.json. Now it's not necesarry